test/box/tx_man.test.lua

```diff
@@ -282,6 +282,12 @@ s:select{}
 
 s:drop()
 
+s = box.schema.space.create('test')
+i = s:create_index('pk')
+s:replace{1}
+collectgarbage('collect')
```

Comment:
Would be nice too see more tests, especially involving DDL operations (which are mentioned in commit message). Otherwise LGTM

Reply:
Just in case - s:drop below is DDL.